lbry/tests/integration/test_claim_commands.py

```diff
@@ -49,6 +49,24 @@ def setUp(self):
             await self.on_transaction_dict(tx)
         return tx
 
+    async def update_stream_to_image_type(self, claim_id, confirm=True):
+        with tempfile.NamedTemporaryFile(suffix='.png') as file:
+            file.write(unhexlify(
+                b'89504e470d0a1a0a0000000d49484452000000050000000708020000004fc'
+                b'510b9000000097048597300000b1300000b1301009a9c1800000015494441'
+                b'5408d763fcffff3f031260624005d4e603004c45030b5286e9ea000000004'
+                b'9454e44ae426082'
+            ))
+            file.flush()
+            tx = await self.out(self.daemon.jsonrpc_stream_update(
+                claim_id, file_path=file.name
+            ))
+        if confirm:
+            await self.on_transaction_dict(tx)
+            await self.generate(1)
+            await self.on_transaction_dict(tx)
+        return tx
+
```

Comment:
With addition of a 3rd method I think it's a good idea to cleanup the pattern here so we don't have 3 methods that basically duplicate 90% of functionality.

Reply:
Absolutely agreed.